src/notifications.js

```diff
@@ -12,7 +12,7 @@ class Notifications extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    const {notifications} = nextProps;
+    const notifications = nextProps.notifications.toJS();
```

Comment:
hey @zeevl ,

What happens if I'm not using immutable-js? This would fail right?
As native arrays do not have `toJs` method?

I have a feeling that if we merge this in then it will break all the existing apps using the lib?
Could be that I need more ☕️